### PR TITLE
bump sentry-cli 1.69.1

### PR DESCRIPTION
**Pull request by bruno-garcia**
Includes a fix needed by the Dart/Flutter SDK:

https://github.com/getsentry/sentry-dart/issues/591#issuecomment-930388078

### CODE REVIEW

**Comment by cortinico** [comment]
In order to bump the CLI you would need to checkout the project, run any gradle command and update the hashes. Similarly to https://github.com/getsentry/sentry-android-gradle-plugin/pull/133/files

**Comment by bruno-garcia** [comment]
Oh that's right. I haven't done that in a couple of years 😁

**Comment by bruno-garcia** [comment]
I'll be away for 20 days so hopefully someone bumps this before I get back

**Review by cortinico — APPROVED** [review]

**Comment by cortinico** [comment]
Looks like there is something going on with the latest release of the CLI?

```
 /tmp/.sentry-cli5131910629045199198.exe: 1: ELF: not found
/tmp/.sentry-cli5131910629045199198.exe: 1: Â: not found
/tmp/.sentry-cli5131910629045199198.exe: 2: 0Ê: not found
/tmp/.sentry-cli5131910629045199198.exe: 3: ¨D: not found
/tmp/.sentry-cli5131910629045199198.exe: 1: Â: not found
/tmp/.sentry-cli5131910629045199198.exe: 2: : not found
/tmp/.sentry-cli5131910629045199198.exe: 5: ÂªÂ: not found
/tmp/.sentry-cli5131910629045199198.exe: 1: Âª: not found
/tmp/.sentry-cli5131910629045199198.exe: 6: :¸: not found
/tmp/.sentry-cli5131910629045199198.exe: 6: Syntax error: Unterminated quoted string
/tmp/.sentry-cli7629879565625909005.exe: 1: Âª: not found
/tmp/.sentry-cli7629879565625909005.exe: 1: ELF: not found
/tmp/.sentry-cli7629879565625909005.exe: 1: Â: not found
/tmp/.sentry-cli7629879565625909005.exe: 2: 0Ê: not found
/tmp/.sentry-cli7629879565625909005.exe: 3: ¨D: not found
/tmp/.sentry-cli7629879565625909005.exe: 1: Â: not found
/tmp/.sentry-cli7629879565625909005.exe: 2: : not found
/tmp/.sentry-cli7629879565625909005.exe: 5: ÂªÂ: not found
/tmp/.sentry-cli7629879565625909005.exe: 1: Âª: not found
/tmp/.sentry-cli7629879565625909005.exe: 6: :¸: not found
/tmp/.sentry-cli7629879565625909005.exe: 6: Syntax error: Unterminated quoted string
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 * Bump: AGP to 7.0.1 (#181)
+* Bump sentry-cli 1.69.0 which includes a fix for Dart debug symbols (#191)
```
Comment (marandaneto) [inline CHANGELOG.md:6]:
```suggestion
* Bump sentry-cli 1.69.1 which includes a fix for Dart debug symbols (#191)
```

**Comment by marandaneto** [comment]
changed to 1.69.1, since 1.69.0 is broken, have to update the checksum now

**Comment by marandaneto** [comment]
> Looks like there is something going on with the latest release of the CLI?
> 
> ```
>  /tmp/.sentry-cli5131910629045199198.exe: 1: �ELF����: not found
> /tmp/.sentry-cli5131910629045199198.exe: 1: Â: not found
> /tmp/.sentry-cli5131910629045199198.exe: 2: �0Ê: not found
> /tmp/.sentry-cli5131910629045199198.exe: 3: ¨D�: not found
> /tmp/.sentry-cli5131910629045199198.exe: 1: Â: not found
> /tmp/.sentry-cli5131910629045199198.exe: 2: �: not found
> /tmp/.sentry-cli5131910629045199198.exe: 5: �ÂªÂ: not found
> /tmp/.sentry-cli5131910629045199198.exe: 1: Âª: not found
> /tmp/.sentry-cli5131910629045199198.exe: 6: �:¸�: not found
> /tmp/.sentry-cli5131910629045199198.exe: 6: Syntax error: Unterminated quoted string
> /tmp/.sentry-cli7629879565625909005.exe: 1: Âª: not found
> /tmp/.sentry-cli7629879565625909005.exe: 1: �ELF����: not found
> /tmp/.sentry-cli7629879565625909005.exe: 1: Â: not found
> /tmp/.sentry-cli7629879565625909005.exe: 2: �0Ê: not found
> /tmp/.sentry-cli7629879565625909005.exe: 3: ¨D�: not found
> /tmp/.sentry-cli7629879565625909005.exe: 1: Â: not found
> /tmp/.sentry-cli7629879565625909005.exe: 2: �: not found
> /tmp/.sentry-cli7629879565625909005.exe: 5: �ÂªÂ: not found
> /tmp/.sentry-cli7629879565625909005.exe: 1: Âª: not found
> /tmp/.sentry-cli7629879565625909005.exe: 6: �:¸�: not found
> /tmp/.sentry-cli7629879565625909005.exe: 6: Syntax error: Unterminated quoted string
> ```

yep, 1.69.1 fixed it

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
looks like 1.69.1 is also broken, have to wait for a new release

**Comment by cortinico** [comment]
> looks like 1.69.1 is also broken, have to wait for a new release

Was this not the case?

**Comment by marandaneto** [comment]
@cortinico the uploaded artifacts were broken on GH artifacts, but we've fixed them manually (sentry-cli repo), so if u download them again, they are good.

**Pull request merged**